.github/workflows/build-java-v2.yaml

```diff
@@ -24,11 +24,6 @@ on:
         description: "This tag will be used in case we have a specific base image tag (now watchtower and label-engine only)"
```

Comment:
I see that at the end of this workflow you want to call the tagging process.
This is then a sort of comprehensive workflow from build to check image to test to tag. It's almost a "main" with sub-methods / steps rights? 
In this case we can not keep the name "build" for the whole file, because build is only a subtask of a higher process.
Can you try to make clear following points:
- what we are doing in each step here (should be understandable by reading the name of each step and possible composite actions that are called -> they also need to be clearly named)
- how should this file be properly named

Inspirations:
https://www.redhat.com/en/topics/devops/what-is-ci-cd?cicd=32h281b

Reply:
The idea is we update the build process to have e2e test check and auto-tagging